### DISCUSSION TITLE
Is it possible to disable a module level mark for a specific test?

### DISCUSSION THREAD

**Discussion by ubalklen** [pytest-dev/pytest#10239] [General] [answered]
I'm using [module level marks](https://docs.pytest.org/en/latest/example/markers.html#marking-whole-classes-or-modules) (`pytestmark`). But now, only one test cannot have this mark. Is there a way to "turn off" this module mark for a single test only (besides getting rid of the module mark and adopting individual marks)?

For example:
```
import pytest

pytestmark = [pytest.mark.vcr]

# This should have the pytest.mark.vcr mark...
def test_a():
    pass

# ...this too...
def test_b():
    pass

# ...but not this one.
def test_z():
    pass
```

Note that this specific mark is handled by the [pytest-recording](https://github.com/kiwicom/pytest-recording) plugin, so I can't ignore the mark for `test_z` in another place (such as a fixture).

**Comment by RonnyPfannschmidt** [comment]
This is not supported and its not a desirable feature in general as it painfully complicates details

**Comment by nicoddemus** [comment, marked as answer]
A quick fix is to move the tests that need the mark to a class, then apply a class-level mark instead.
